sound/hybrid_font.h

```diff
@@ -205,6 +205,7 @@ class HybridFont : public SaberBase, public Looper {
   RefPtr<BufferedWavPlayer> next_hum_player_;
   RefPtr<BufferedWavPlayer> swing_player_;
   RefPtr<BufferedWavPlayer> lock_player_;
+  RefPtr<BufferedWavPlayer> hum_player_;
```

Comment:
Does this work? Because there is already a hum_player_ on line 204...

Reply:
oops. that's leftover from when there was still an idle_player sorry. 
Don't know if it works, my local copy has 208 blank.
Removed.

Reply:
> Don't know if it works, my local copy has 208 blank.
So does it work now?
I don't want to review code that doesn't work.

Reply:
Oh, sorry that was unclear.
You asked "Does this work" and my reply was supposed to convey that
my local copy did not have it when successfully testing. So I can't say if it works with a redundant hum_player, as i hadn't run the code with it present.
Removed that line and tested. Works well.